internal/core/src/mmap/Column.h

```diff
@@ -123,6 +128,7 @@ class ColumnBase {
         AssertInfo(data_ != MAP_FAILED,
                    "failed to create file-backed map, err: {}",
                    strerror(errno));
+        madvise(data_, mapped_size, MADV_WILLNEED);
```

Comment:
madvise here may lead to memory increase. Perhaps it's best not to add it for now.

Reply:
ack, will remove it before merging.